qcodes/instrument/parameter.py

```diff
@@ -1763,73 +1804,36 @@ class GetLatest(DelegateAttributes):
 
     Args:
         parameter: Parameter to be wrapped.
-        max_val_age: The max time (in seconds) to trust a
-            saved value obtained from get_latest(). If this parameter has not
-            been set or measured more recently than this, perform an
-            additional measurement.
     """
-    def __init__(self, parameter: _BaseParameter,
-                 max_val_age: Optional[Number] = None):
+    def __init__(self, parameter: _BaseParameter):
         self.parameter = parameter
-        self.max_val_age = max_val_age
 
     delegate_attr_objects = ['parameter']
     omit_delegate_attrs = ['set']
 
     def get(self) -> ParamDataType:
-        """Return latest value if time since get was less than
+        """
```

Comment:
At some point should add deprecation warning to this  I guess

Reply:
indeed, that's the plan. but in another PR in the future :)